pkg/syncer/syncer.go

```diff
@@ -394,6 +395,14 @@ func StartSyncer(ctx context.Context, cfg *SyncerConfig, numSyncerThreads int, i
 		go startSyncerTunnel(ctx, upstreamConfig, downstreamConfig, logicalcluster.From(syncTarget), cfg.SyncTargetName)
 	}
 
+	StartHeartbeatKeeper(ctx, kcpSyncTargetClient, cfg.SyncTargetName, cfg.SyncTargetUID)
+
+	return nil
+}
+
+func StartHeartbeatKeeper(ctx context.Context, kcpSyncTargetClient kcpclientset.Interface, syncTargetName, syncTargetUID string) {
```

Comment:
what's a keeper?
![image](https://user-images.githubusercontent.com/730123/217200348-fccd9b38-12b1-49ac-9305-50c47283ee00.png)

Reply:
It keeps the heartbeat up-to-date. But I agree it's probably a bad naming.
Changing it to simply `StartHeartBeat()`

Reply:
Fixed in commit 13a4a065c80f0bf3815d69defc0ffcd0728fce2d